charts/external-dns/templates/deployment.yaml

```diff
@@ -124,8 +124,25 @@ spec:
             - --managed-record-types={{ . }}
             {{- end }}
             - --provider={{ $providerName }}
-          {{- range .Values.extraArgs }}
+          {{- if kindIs "map" .Values.extraArgs }}
+            {{- range $key, $value := .Values.extraArgs }}
+            {{- if not (kindIs "invalid" $value) }}
+              {{- if kindIs "slice" $value }}
+                {{- range $value }}
+            - --{{ $key }}={{ tpl (. | toString) $ }}
+                {{- end }}
+              {{- else }}
+            - --{{ $key }}={{ tpl ($value | toString) $ }}
+              {{- end }}
+            {{- else }}
+            - --{{ $key }}
+            {{- end }}
+            {{- end }}
+          {{- end }}
+          {{- if kindIs "slice" .Values.extraArgs }}
+            {{- range .Values.extraArgs }}
             - {{ tpl . $ }}
+            {{- end }}
```

Comment:
As a convention in this chart we keep template directives at the same indentation level as their parent. Could you please follow this pattern here?

Reply:
@stevehipwell so you mean no indentation of any of the loops and conditionals then?